CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [2.2.0] - 2022-01-24
+### Changed
+- Added telemetry feature.
```

Comment:
For consistency with the Node.js SDK let's use this snippet:  `Added SDK telemetry data collection`